src/idl_gen_swift.cpp

```diff
@@ -1158,44 +1188,81 @@ class SwiftGenerator : public BaseGenerator {
 
   void GenEnum(const EnumDef &enum_def) {
     if (enum_def.generated) return;
+    const bool is_bit_flags =
+        enum_def.attributes.Lookup("bit_flags") != nullptr;
+    const bool is_legacy_naming =
+        enum_def.attributes.Lookup("swift_no_legacy_names") == nullptr;
```

Comment:
@dbaileychess do we need to add the new attribute somewhere? or would it be even needed?

Reply:
Reminds me -- is there a way to specify a "global" attribute?  I'd like to be able to globally set some attributes that just apply to everything, to avoid having to put this on every enum.

Reply:
attributes should be added to Parser class in idl.h